lib/src/visitor/async_evaluate.dart

```diff
@@ -1344,7 +1344,9 @@ class _EvaluateVisitor
                     if (!variable.isGuarded) variable.name
                 });
 
-      _assertConfigurationIsEmpty(newConfiguration);
+      if (node == _stylesheet.forwards.last) {
+        _assertConfigurationIsEmpty(newConfiguration);
```

Comment:
I'm not sure this is right either... since `newConfiguration` contains all configured variables that were passed into the current module, it might still have an unsatisfied configuration for the *current* module. Really we just want to verify that there all the variables configured by `node.configuration` were removed.